### PR TITLE
Update generic.md

### LINKED ISSUE #1582: Remove unnecessary spaces generic.md

### PR DESCRIPTION
**Pull request by krishdevdb**
closes #1582 and update the grammar :)

#### What does this PR do?
Fix #1582 

#### Description of Task to be completed?
- Remove Unnecessary Spaces In `generic.md`
- Fix Grammar, by refactoring the text

#### How can this be manually tested?
- Read it 🤔 

#### Any background context you want to provide?
- Found it via `#good-first-issue` channel on discord

#### What is the relevant issue link?
#1582

### CODE REVIEW

**Review by starlightknown — CHANGES_REQUESTED** [review]

**Review by starlightknown — DISMISSED** [review]
Looks good 👍 thank you

**Comment by krishdevdb** [comment]
Thank You For The 🟩

File: youtube/generic.md [hunk 1]
```diff
@@ -1,37 +1,46 @@
...
+Join our inclusive open source community EddieHub now for **free** at http://eddiehub.org
+
+Make sure to checkout my website and sign up to our community **newsletter**: http://eddiejaoude.io for my live stream + video schedule and community calls
+
+Join Our **Community GitHub Organisation**:
+
+- Go to https://github.com/EddieHubCommunity/support/issues/choose 
+- Choose The Issue Type Of **Invite Me To The Organisation**
+- Open The Issue
+- You get a invite to the community by our GitHub Action 🎉
```
Comment (schmelto) [inline youtube/generic.md:19]:
```suggestion
- You get an invitation to the community by our GitHub Action 🎉
```

**Review by starlightknown — APPROVED** [review]
LGTM 👍🏼

File: youtube/generic.md [hunk 2]
```diff
@@ -1,37 +1,46 @@
-...
+## Youtube Channel
+Don’t forget to support my channel by subscribing it! It’s free: https://www.youtube.com/eddiejaoude?sub_confirmation=1
```
Comment (Vyvy-vi) [inline youtube/generic.md:2]:
```suggestion
Don’t forget to support the channel by subscribing it! It’s free: https://www.youtube.com/eddiejaoude?sub_confirmation=1
```
I think phrasing it like this is more humble?

File: youtube/generic.md [hunk 3]
```diff
@@ -1,37 +1,46 @@
...
-https://www.youtube.com/eddiejaoudetv
+**Make sure to tell your friends about it, so that they also learn something new!**
 
-JOIN our inclusive open source community EddieHub now for FREE ... http://eddiehub.org, also checkout my website and sign up to our community NEWSLETTER... http://eddiejaoude.io for my live stream + video schedule and community calls
+## Youtube Shorts
+I have another youtube channel for short clips, please **take a moment subscribing it: https://www.youtube.com/eddiejaoudetv**
 
-Alslo join our community GitHub organisation by creating an issue with the type "invite me to the organisation" :)
+## Join Our Community
+Join our inclusive open source community EddieHub now for **free** at http://eddiehub.org
```
Comment (Vyvy-vi) [inline youtube/generic.md:10]:
```suggestion
Join our inclusive open source community EddieHub now for **free** at https://eddiehub.org
```
I think that site has `https`. Linking to secure links is generally better

File: youtube/generic.md [hunk 4]
```diff
@@ -1,37 +1,46 @@
...
+
+Make sure to checkout my website and sign up to our community **newsletter**: http://eddiejaoude.io for my live stream + video schedule and community calls
+
+Join Our **Community GitHub Organisation**:
+
+- Go to https://github.com/EddieHubCommunity/support/issues/choose
```
Comment (Vyvy-vi) [inline youtube/generic.md:16]:
```suggestion
- Go to https://github.com/EddieHubCommunity/support/issues/new/choose 
```
I think the `choose` route is under `new`
OR Maybe we can link to the `Invite Me to the org` template directly? -> https://github.com/EddieHubCommunity/support/issues/new?assignees=&labels=invite+me+to+the+organisation&template=invitation.md&title=Please+invite+me+to+the+GitHub+Community+Organization

File: youtube/generic.md [hunk 5]
```diff
@@ -1,37 +1,46 @@
...
+- You get an invitation to the community by our GitHub Action 🎉
 
 ---------------------------------------------------------------------------------------------------
 
-A BIG thank you to my GitHub Sponsors:
-***
-Stephen Mount http://github.com/stemount
-Nicholas Carrigan https://github.com/nhcarrigan
-***
+## Thanks To Sponsors
```
Comment (Vyvy-vi) [inline youtube/generic.md:23]: Maybe this can be a **BIG** thank you?

File: youtube/generic.md [hunk 6]
```diff
@@ -1,37 +1,46 @@
...
 - Andrew Cunliffe https://github.com/andrew-cunliffe
 - Alexander Stoichkov https://github.com/SashoStoichkov
 - Allan Regush https://github.com/AllanRegush
 
 ---------------------------------------------------------------------------------------------------
 
-Don’t forget to support my channel by subscribing below, it’s free, and also share with your friends. Subscribe now!!
-https://www.youtube.com/eddiejaoude?sub_confirmation=1
+## Social Media
+Follow My **Social Media For Behind The Scenes Footage**
```
Comment (Vyvy-vi) [inline youtube/generic.md:35]:
```suggestion
Follow on socials for behind the scenes footage and more... Join discord to continue the conversation...
```
In my opinion, the older phrasing sounded a bit better than the boldified "Social media for being the scenes footage". Moreover, the socials have more stuff than just behind the scenes

**Review by Vyvy-vi — CHANGES_REQUESTED** [review]
I had a question, does the markdown bold - `**text**`, work on YouTube's description?

**Comment by starlightknown** [comment]
> I had a question, does the markdown bold - `**text**`, work on YouTube's description?

Nope it doesn't

File: youtube/generic.md (hunk 3 above)
Comment (schmelto) [reply youtube/generic.md:10]: Nope https doesn't work here 😐

**Comment by Vyvy-vi** [comment]
> > I had a question, does the markdown bold - `**text**`, work on YouTube's description?
> 
> Nope it doesn't

Then, maybe it doesn't make much sense to use them here, since Eddie would be copying this file and pasting it as a description?

**Comment by krishdevdb** [comment]
Eddie was using some words in caps which is not a perfect way of visual difference. I have made all the content with some ui skills suitable in mind. It doesn't work on youtube but atleast people can see it on GitHub. For the copying problem of the asterisks, Eddie can copy the preview instead

File: youtube/generic.md [hunk 7]
```diff
@@ -1,37 +1,46 @@
...
 
-Follow on other socials for behind the scenes footage, join discord to continue the conversation...
+- Website: http://eddiejaoude.io
```
Comment (Vyvy-vi) [inline youtube/generic.md:37]: Maybe we should use `https` links?

- Website: https://www.eddiejaoude.io 
- GitHub: https://github.com/eddiejaoude
- Youtube: https://youtube.com/eddiejaoude
- Twitter: https://twitter.com/eddiejaoude
- Instagram: https://instagram.com/eddiejaoude
- Linkedin: https://linkedin.com/in/eddiejaoude
- GitHub Community Org: https://github.com/EddieHubCommunity

**Review by Vyvy-vi — CHANGES_REQUESTED** [review]
I left a few comments for the PR :)

**Comment by krishdevdb** [comment]
> I left a few comments for the PR :)

Fixed all of them

File: youtube/generic.md [hunk 8]
```diff
@@ -34,13 +34,13 @@ Join Our **Community GitHub Organisation**:
 ## Social Media
 Follow on socials for behind the scenes footage and more.
 
-- Website: http://eddiejaoude.io 
+- Website: https://eddiejaoude.io
```
Comment (deephunt3r) [inline youtube/generic.md:37]: @krishdevdb oops, this is the link -
https://www.eddiehub.org as mentioned by @Vyvy-vi earlier

**Review by deephunt3r — CHANGES_REQUESTED** [review]

File: youtube/generic.md (hunk 8 above)
Comment (krishdevdb) [reply youtube/generic.md:37]: I guess I forgot this 😅 Fixing It

File: youtube/generic.md [hunk 9]
```diff
@@ -7,9 +7,9 @@ Don’t forget to support my channel by subscribing it! It’s free: https://www
 I have another youtube channel for short clips, please **take a moment subscribing it: https://www.youtube.com/eddiejaoudetv**
 
 ## Join Our Community
-Join our inclusive open source community EddieHub now for **free** at http://eddiehub.org
+Join our inclusive open source community EddieHub now for **free** at https://eddiehub.org
```
Comment (deephunt3r) [inline youtube/generic.md:10]: @krishdevdb the above link too cintains *www*

**Review by deephunt3r — CHANGES_REQUESTED** [review]

File: youtube/generic.md (hunk 9 above)
Comment (deephunt3r) [reply youtube/generic.md:10]: > @krishdevdb the above link too cintains _www_

@krishdevdb better change every link https://eddiehub.org in the doc to https://www.eddiehub.org/

File: youtube/generic.md [hunk 10]
```diff
@@ -22,25 +22,25 @@ Join Our **Community GitHub Organisation**:
...
+- GitHub: https://www.github.com/eddiejaoude
+- Youtube: https://www.youtube.com/eddiejaoude
+- Twitter: https://www.twitter.com/eddiejaoude
+- Instagram: https://www.instagram.com/eddiejaoude
+- Linkedin: https://www.linkedin.com/in/eddiejaoude
+- GitHub Community Org: https://www.github.com/EddieHubCommunity
 
 ## Next
-Join Our Conversation At Discord: https://discord.eddiehub.org/
+Join Our Conversation At Discord: https://www.discord.eddiehub.org/
```
Comment (deephunt3r) [inline youtube/generic.md:46]: @krishdevdb The discord link doesn't contains _www_

**Review by deephunt3r — CHANGES_REQUESTED** [review]

**Review by deephunt3r — DISMISSED** [review]
LGTM 🚀

File: youtube/generic.md [hunk 11]
```diff
@@ -1,37 +1,46 @@
...
+- You get an invitation to the community by our GitHub Action 🎉
 
 ---------------------------------------------------------------------------------------------------
 
-A BIG thank you to my GitHub Sponsors:
-***
-Stephen Mount http://github.com/stemount
-Nicholas Carrigan https://github.com/nhcarrigan
-***
+## Big Thanks To Sponsors
```
Comment (Vyvy-vi) [inline youtube/generic.md:23]: Perhaps we can mention `Github Sponsors`, in order to convey the platform of sponsorship?

File: youtube/generic.md [hunk 12]
```diff
@@ -1,37 +1,46 @@
...
 ---------------------------------------------------------------------------------------------------
 
-Don’t forget to support my channel by subscribing below, it’s free, and also share with your friends. Subscribe now!!
-https://www.youtube.com/eddiejaoude?sub_confirmation=1
+## Social Media
+Follow on socials for behind the scenes footage and more.
 
-Follow on other socials for behind the scenes footage, join discord to continue the conversation...
+- Website: https://www.eddiejaoude.io 
+- GitHub: https://www.github.com/eddiejaoude
```
Comment (Vyvy-vi) [inline youtube/generic.md:38]: I'm not sure if we need the `www` for all the links(except `https://www.eddiehub.org` and `https://www.eddiejaoude.io/`). Linking directly via the domain tends to be more shorter.
(thoughts?)

**Review by Vyvy-vi — CHANGES_REQUESTED** [review]

File: youtube/generic.md (hunk 12 above)
Comment (krishdevdb) [reply youtube/generic.md:37]: I think someone asked me turn all the links into www

**Comment by krishdevdb** [comment]
Pull Request Closed, Due to one mistake I made with the fork 😓 New Version #1795

**Pull request closed without merging**